fuse/nodefs/dir.go

```diff
@@ -13,33 +13,30 @@ import (
 
 type connectorDir struct {
 	node  Node
+	inode *Inode
 	rawFS fuse.RawFileSystem
 
 	// Protect stream and lastOffset.  These are written in case
```

Comment:
There is also a remaining reference to `lastOffset`. We might want to update that comment aswell.